vars/tasks.groovy

```diff
@@ -156,13 +156,14 @@ def functionalXpack(Map params = [:]) {
       }
     }
 
-    whenChanged([
-      'x-pack/plugins/synthetics/',
-    ]) {
-      if (githubPr.isPr()) {
-        task(kibanaPipeline.functionalTestProcess('xpack-UptimePlaywright', './test/scripts/jenkins_uptime_playwright.sh'))
-      }
-    }
+    // Skipped, process is hanging https://github.com/elastic/kibana/issues/137185
+    // whenChanged([
```

Comment:
I don't believe anything uses the old Jenkins config.

For Buildkite your looking for https://github.com/elastic/kibana/blob/main/.buildkite/scripts/pipelines/pull_request/pipeline.ts#L118

Reply:
You can see the script location as part of the task

![image](https://user-images.githubusercontent.com/40265/181022301-897e7ffa-eafb-4a84-8bd3-dc0a4ae88f9a.png)

If it's too long and gets cut off, you can look at the environment tab as it's represented in full under `BUILDKITE_SCRIPT_PATH`